src/styles/pages/blogPost.scss

```diff
@@ -76,6 +76,7 @@
 	}
 
 	&thumbnail {
+		width: 100%;
```

Comment:
What does this change?

Reply:
This fixes the image alignment. Without this the blog post images appear aligned to the left which looks bad and inconsistent.

Reply:
Sounds good to me